README.md

```diff
@@ -335,84 +139,23 @@ compiler crashes.
 
 ### Build Failures
 
-Make sure you are using the [correct release](#macos) of Xcode.
+Try the suggestions in
+[Troubleshooting build issues](/docs/HowToGuides/GettingStarted.md#troubleshooting-build-issues).
+
+Make sure you are using the
+[correct release](/docs/HowToGuides/GettingStared.md#installing-dependencies)
+of Xcode.
 
 If you have changed Xcode versions but still encounter errors that appear to
 be related to the Xcode version, try passing `--clean` to `build-script`.
 
 When a new version of Xcode is released, you can update your build without
 recompiling the entire project by passing the `--reconfigure` option.
```

Comment:
TIL. Since `--reconfigure` does not show up in the `build-script --help` output, maybe this sentence should also say "to `build-script"`.